malariagen_data/anoph/genome_features.py

```diff
@@ -0,0 +1,422 @@
+from typing import Dict, Literal, Optional, Sequence, Tuple, Union
+
+import bokeh.models
+import bokeh.plotting
+import numpy as np
+import pandas as pd
+from numpydoc_decorator import doc
+from typing_extensions import Annotated, TypeAlias
+
+from ..util import Region, read_gff3, resolve_region, unpack_gff3_attributes
+from .base import DEFAULT, base_params
+from .genome_sequence import AnophelesGenomeSequenceData
+
+
+class gplt_params:
```

Comment:
Moving these parameter definitions here, as this is the first time they're needed.